ethpm_cli/manifest.py

```diff
@@ -76,6 +75,168 @@ def generate_custom_manifest(project_dir: Path) -> None:
     )
 
 
+def amend_manifest(manifest_path: Path) -> None:
+    cli_logger.info("Manifest Wizard")
+    cli_logger.info("---------------")
+    cli_logger.info("Amend a local manifest.")
+    cli_logger.info("")
+
+    manifest = json.loads(manifest_path.read_text())
+    validate_manifest_against_schema(manifest)
+    pkg_repr = f"<Package {manifest['package_name']}=={manifest['version']}>"
+    cli_logger.info(f"Valid manifest for {pkg_repr} found at {manifest_path}.")
+    builder_fns = (
+        amend_description(manifest),
+        amend_license(manifest),
+        amend_authors(manifest),
+        amend_keywords(manifest),
+        amend_links(manifest),
+        *amend_deployments(manifest),
+        gen_validate_manifest(),
+    )
+    final_fns = (fn for fn in builder_fns if fn is not None)
```

Comment:
Maybe instead of having the functions return `None` they could just return a `noop` function that did nothing.

Reply:
To eliminate having to do this entire comprehension and feed the `builder_fns` straight into the build on line 98? I tested with a simple noop and it'll break the manifest builder, so filtering out the `noop`s / `None` will still be necessary - or I can add a special `noop` to `ethpm`'s `builder`

Reply:
Not *just* to eliminate the comprehension, but more to remove the need for all those functions to have `Optional` return types.  Homogeneous typing such that we don't have to have runtime filters like this can be a nice improvement and helps reduce complexity (this comprehension and the `Optional` return types are a type of complexity).

> or I can add a special `noop`

Yes, wasn't necessarily meaning `noop = lambda x: None` but whatever a `noop` would mean in the context of the builder pipeline, a function that does nothing in the pipeline.